### DISCUSSION TITLE
After SatPI update no stream data no picture, with Inverto IDLU-UWT110CUO10-32P Multiswitch

### DISCUSSION THREAD

**Discussion by lordduckstein** [Barracuda09/SATPI#209] [Q&A] [answered]
Hello,

after the update. The SATIP stream is broken. I find the tuners in DVBviewer but there is no picture. Before the update the system were running.

Here my system
- GigablueUHD UE 4K, 2 x Sat FBC-Tuner,
- Inverto IDLU-UWT110CUO10-32P Multiswitch

Here my log 
[log.txt](https://github.com/Barracuda09/SATPI/files/14128424/log.txt)

![Fritzbox Hardware](https://github.com/Barracuda09/SATPI/assets/30575840/f2df17f2-09e0-43c8-a07f-cc85562a4cbc)
![Fronted Overview](https://github.com/Barracuda09/SATPI/assets/30575840/87b7ff18-937d-4ff1-bbc3-f52af3c378d2)
![Fonted Info Part1](https://github.com/Barracuda09/SATPI/assets/30575840/4925aa1a-ea48-4dec-aa51-40c964d7d883)
![Fonted Info Part2](https://github.com/Barracuda09/SATPI/assets/30575840/bdb74551-d5db-456e-98d5-1c8a964f5110)
![Fonted Info Part3](https://github.com/Barracuda09/SATPI/assets/30575840/2d50084c-59a1-4f8d-bf4f-6dc7a7845c16)
![Fonted Info Part4](https://github.com/Barracuda09/SATPI/assets/30575840/453491ce-0d76-4c01-a4af-309043523686)

The fronted 2 is not configured, because I use It in Enigma2. Ich have OpenATV image with the new Devel-Image 7.5 but in other images 7.3 or 7.4 it is also not working, after the update. Guess before the update all were running with same settings!

I think it must issues after the last changes ...

Thank you
Lordi

**Comment by Barracuda09** [comment]
Hi @lordduckstein 

Thank you for your feedback. It seems that you do not get a lock to the requested frequency. 
I have a few questions:
- Did it work before then?
- What kind of LNB is connected to the multiswitch?
- Did you adjust the `Repeat DiSEqC` to more then 1?

```
  Thu Feb  1 17:41:53.0191 2024  6  Frontend: 1, Not locked yet   (FE status 0x01)...   <--- *** No Lock ***
  Thu Feb  1 17:41:52.5683 2024  6  Frontend: 1, Tuned, waiting on lock...
  Thu Feb  1 17:41:52.5682 2024  7  Frontend: 1, Set Properties: Frequency 1420000
  Thu Feb  1 17:41:52.5682 2024  6  Frontend: 1, Closing /dev/dvb/adapter0/frontend0 with fd: 12
  Thu Feb  1 17:41:52.4398 2024  6  Frontend: 1, Sending DiSEqC: Command Repeated
  Thu Feb  1 17:41:52.3166 2024  6  Frontend: 1, Sending DiSEqC: Command Repeated
  Thu Feb  1 17:41:52.1934 2024  6  Frontend: 1, Sending DiSEqC: Command Repeated
  Thu Feb  1 17:41:52.0701 2024  6  Frontend: 1, Sending DiSEqC: Command Repeated
  Thu Feb  1 17:41:51.9468 2024  6  Frontend: 1, Sending DiSEqC: Command Repeated
  Thu Feb  1 17:41:51.8235 2024  6  Frontend: 1, Sending DiSEqC: Command Repeated
  Thu Feb  1 17:41:51.7002 2024  6  Frontend: 1, Sending DiSEqC: Command Repeated
  Thu Feb  1 17:41:51.6767 2024  6  HTTP Client 192.168.178.44:56891 Connection closed with fd: 9
  Thu Feb  1 17:41:51.5765 2024  6  Frontend: 1, Sending DiSEqC: Command Repeated
  Thu Feb  1 17:41:51.5533 2024  6  Frontend: 1, Sending DiSEqC: [0x70] [0x05] [0x34] [0x03] - DiSEqC Src: 0 - UB: 0
  Thu Feb  1 17:41:51.5533 2024  7  Frontend: 1, Using LNB with: Lof-High=10600 MHz  Lof-Low=9750 MHz  Lof-Switch=11700 MHz -> IF=1432 MHz
  Thu Feb  1 17:41:51.5082 2024  6  Frontend: 1, Opened /dev/dvb/adapter0/frontend0 for Writing DiSEqC command with fd: 12
  Thu Feb  1 17:41:51.5079 2024  6  Frontend: 1, Start tuning process for DVB-S(2)...
  Thu Feb  1 17:41:51.5078 2024  6  Frontend: 1, Opened /dev/dvb/adapter0/frontend0 for Read/Write with fd: 11 (0 ms)
  Thu Feb  1 17:41:51.4574 2024  6  Frontend: 1, Closing all active PID filters...
  Thu Feb  1 17:41:51.4574 2024  6  Frontend: 1, Updating frontend...
  Thu Feb  1 17:41:51.4573 2024  7  Frontend: 1, Parsing transport parameters (Finished)
  Thu Feb  1 17:41:51.4572 2024  7  Frontend: 1, Parsing PID parameter: pids=0
  Thu Feb  1 17:41:51.4570 2024  6  Frontend: 1, New frequency requested (0.0000 -> 12032.0000), clearing old channel data...
  Thu Feb  1 17:41:51.4569 2024  6  Frontend: 1, Parsing transport parameters...
  Thu Feb  1 17:41:51.4569 2024  6  Frontend: 1, StreamClient with SessionID 0263094224 for dvbs2
  Thu Feb  1 17:41:51.4567 2024  7  Frontend: 1, Found Streaming type: RTP/AVP
  Thu Feb  1 17:41:51.4566 2024  6  Found FrondtendID: x (fe=x)  StreamID: x  SessionID: 0263094224  New Session: true
  Thu Feb  1 17:41:51.4561 2024  7  <CRLF>
  Thu Feb  1 17:41:51.4561 2024  7  Transport: RTP/AVP;unicast;client_port=47742-47743
  Thu Feb  1 17:41:51.4561 2024  7  CSeq: 2
  Thu Feb  1 17:41:51.4561 2024  7  RTSP Stream data from client None with IP 192.168.178.44 on Port 56892: SETUP rtsp://192.168.178.58:554/?src=1&freq=12032&msys=dvbs2&plts=off&fec=910&pol=h&ro=0.35&sr=27500&mtype=qpsk&pids=0 RTSP/1.0
  Thu Feb  1 17:41:51.4551 2024  7  <CRLF>
  Thu Feb  1 17:41:51.4551 2024  7  Public: OPTIONS, DESCRIBE, SETUP, PLAY, TEARDOWN
```

Kind regards,
Marc

**Comment by lordduckstein** [comment]
Hello Marc,

thanks for your support

1.) Yes It has worked before, only I have update the system. I have made a heavy mistake "never touch an running system".

2.) It is an Durline eco Quad-LNB

3.) which DiSEqC settings do you mean in enigma2?
 
![Tuner B](https://github.com/Barracuda09/SATPI/assets/30575840/3244f45f-646e-4329-a3f1-b0015afa55b7)

There is no DiSEcQ settings

or in SATIP

I have try with DiSEcQ Repeat = 0
![DiSEqC](https://github.com/Barracuda09/SATPI/assets/30575840/c8b54135-0d65-494e-aa3c-d09c2b222754)

or before  with DiSEcQ Repeat = 8


![Fonted Info Part4](https://github.com/Barracuda09/SATPI/assets/30575840/53e25595-a799-474f-8747-83fdacf26d62)

Nothing happens...

Thank you

Lordi

**Comment by lordduckstein** [comment]
Do you have old ipk`s from SATPI then I can figure out it is SATPI or other things like enigma2?

**Comment by Barracuda09** [comment]
> Do you have old ipk`s from SATPI then I can figure out it is SATPI or other things like enigma2?

No I do not have one, but it should work with this version.

I have some further questions:
- Can you give the Enigma2 tuner settings for Tuner A (Like you gave for Tuner B)
- The Channel freq of Unicable II starts at 0 So SCR1 is `Channel Slot (0-32)` is 0
  - Example (See below) CH1 (SCR1) is:
    - `Channel Freq (MHz)` - 1210 
    - `Channel Slot (0-32)` - 0
  - Example (See below) CH8 (SCR8) is:
    - `Channel Freq (MHz)` - 1092 
    - `Channel Slot (0-32)` - 7
```
User band (channel) frequencies | Configurable; Default 32 User bands:
  | CH1: 1210 MHz (EN50494+EN50607)
  | CH2: 1420 MHz (EN50494+EN50607)
  | CH3: 1680 MHz (EN50494+EN50607)
  | CH4: 2040 MHz (EN50494+EN50607)
  | CH5: 984 MHz (EN50494+EN50607)
  | CH6: 1020 MHz (EN50494+EN50607)
  | CH7: 1056 MHz (EN50494+EN50607)
  | CH8: 1092 MHz (EN50494+EN50607)
  | CH9: 1128 MHz (EN50607)
```

**Comment by lordduckstein** [comment]
Hello Marc,

Tuner A is not configured in Enigma2, because It is not work. SATPI or Enigma2 don´t like this, when the tuner is in both are configured

![Tuner A](https://github.com/Barracuda09/SATPI/assets/30575840/cd3feabe-1a10-40ce-a9bd-138bf1cb9188)

Yes I know that we have to start with "0"

Here again the settings for Tuner A in SATPI

![s2](https://github.com/Barracuda09/SATPI/assets/30575840/0d64db55-57c2-4118-9a39-4cf2d6f32f36)


Here the actuell Fronted overview the other tuner I have disabled

![Screenshot 1](https://github.com/Barracuda09/SATPI/assets/30575840/88bb2951-57c8-4b1d-88d8-e8af2d934a1b)

It is not working

Here a new log with only Tuner A is enabled:

[log2.txt](https://github.com/Barracuda09/SATPI/files/14129660/log2.txt)

Lordi

**Comment by Barracuda09** [comment]
I see you use `Channel Freq` of 1420 with `Channel Slot` 0 -> Should that not be set to 1

**Comment by lordduckstein** [comment]
Sorry this was the wrong Picture

![S3](https://github.com/Barracuda09/SATPI/assets/30575840/b09285a7-c724-4e28-98ba-714280b222a8)

this is the right

Did not work

Lordi

**Comment by Barracuda09** [comment]
You could increase 
- `Delay before write` to 15
- `Delay after write` to 30

**Comment by lordduckstein** [comment]
Hello Marc,

no it don´t work 

[log3.txt](https://github.com/Barracuda09/SATPI/files/14129853/log3.txt)

**Comment by Barracuda09** [comment]
Did you try setting it just both at 100

**Comment by lordduckstein** [comment]
Hello Marc,

what should I say your are an "GOD"

IT WORKS

You are my Hero

CU
Lordi

**Comment by Barracuda09** [comment]
@lordduckstein No problem

**Comment by lordduckstein** [comment, marked as answer]
Hello,

this is the solution for my system:

- Gigablue UHD UE 4K, 2 x Sat FBC-Tuner, (receiver), openATV-image, SATPI is installed
- Inverto IDLU-UWT110CUO10-32P Multiswitch (Unicable II)
- Durline Eco blue quattro-LNB

configuration settings

1.) Settings in enigma

![1](https://github.com/Barracuda09/SATPI/assets/30575840/0f19ecd4-1a42-4b82-a012-ee8118a10886)

Only tuner A is configured, the other tuner are disabled

tuner A configuration

![2](https://github.com/Barracuda09/SATPI/assets/30575840/20b6f5b8-4d4c-48b1-8fee-3ce29bf16fc5)

2. Settings in SATIP (It must be installed and running)

Open with browser http://your-ip-receiver:8875

![3](https://github.com/Barracuda09/SATPI/assets/30575840/f3995266-65ed-4715-8d6b-228f38268b88)

Frontend 1 (FBC-Root):

- disabled
  nothing is to do, because this is the Tuner A in enigma2 openATV, when you configured this tuner again
  so enigma2 and SATIP fight for the tuner and nothing work.

Fronted 2 (FBC-Root):

This are my configs

![4](https://github.com/Barracuda09/SATPI/assets/30575840/35dadd2e-145a-4efc-bfde-ceca804e1fa8)
![5](https://github.com/Barracuda09/SATPI/assets/30575840/7e63d60d-c0dd-4af9-a16d-ed2c2d63a20e)
![6](https://github.com/Barracuda09/SATPI/assets/30575840/3a1bd014-add7-4b76-8759-c540c0e24ebe)
![7](https://github.com/Barracuda09/SATPI/assets/30575840/41140a45-6d4e-4ece-a5f0-386bc89696d1)

The other Fronted 3 (FBC) - Fronted 8 (FBC) configs
- Are the same in the pictures before (Fronted 2 (FBC-Root))
- Only the "Channel Freq (MHz)" and "Channel Slot (0-32)" are different

   CH0: 1210 MHz (EN50494+EN50607) <-------------- are used in Enigma Tuner A FBC-Root (there start with channel1)
   CH1: 1420 MHz (EN50494+EN50607) <--------------- Tuner B (Fronted 2) FBC-Root
   CH2: 1680 MHz (EN50494+EN50607) <--------------- Tuner C (Fronted 3) FBC
   CH3: 2040 MHz (EN50494+EN50607) <--------------- Tuner D (Fronted 4) FBC
   CH4: 984 MHz (EN50494+EN50607)   <--------------- Tuner E (Fronted 5) FBC
   CH5: 1020 MHz (EN50494+EN50607) <--------------- Tuner F (Fronted 6) FBC
   CH6: 1056 MHz (EN50494+EN50607) <--------------- Tuner G (Fronted 7) FBC
   CH7: 1092 MHz (EN50494+EN50607) <--------------- Tuner H (Fronted 8) FBC

Thank you Marc for helping

Best regards
Lordi

**Reply by Barracuda09** [reply]
Thanks for your answer!

**Comment by lordduckstein** [comment]
Hello, 

an additional information when you are in mixed operation dvb-c virtuell or dvb-c with dvb-s2 you have to do follwing settings:

- Delay before write: 1000
- Delay after write: 1000

Cu
Lordi
